WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShipping Customs/WooShippingCustomsItem.swift

```diff
@@ -0,0 +1,201 @@
+import SwiftUI
+import WooFoundation
+
+struct WooShippingCustomsItem: View {
+    /// Whether the item list is collapsed
+    @State private var isCollapsed: Bool = true
+    @ObservedObject var viewModel: WooShippingCustomsItemViewModel
+    @State private var isShowingHSTarrifInfoWebView = false
+    @State private var isShowingCountries = false
+
+    var body: some View {
+        CollapsibleView(isCollapsed: $isCollapsed,
+                        shouldShowDividers: false,
+                        backgroundColor: .clear,
+                        label: {
+            VStack(alignment: .leading, spacing: Layout.collapsibleViewVerticalSpacing) {
+                HStack {
+                    Text(viewModel.title)
+                        .headlineStyle()
+                    Spacer()
+                    Image(systemName: "exclamationmark.circle")
+                        .foregroundColor(.withColorStudio(name: .red, shade: .shade60))
+                        .renderedIf(viewModel.informationIsMissing)
+                }
+
+                VStack(alignment: .leading, spacing: Layout.collapsibleViewBottomLabelVerticalSpacing) {
+                    HStack {
+                        Text(viewModel.description)
+                        Spacer()
+                        Text(viewModel.hsTariffNumber)
+                    }
+                    HStack {
+                        Text(viewModel.originCountry.name)
+                        Spacer()
+                        Text(viewModel.weightPerUnit)
+                        Text("•")
+                        Text(viewModel.valuePerUnit)
+                    }
+                }.renderedIf(isCollapsed)
+                    .foregroundColor(.primary)
+                    .subheadlineStyle()
```

Comment:
Is this text supposed to be subheadline style? The design looks like it's using regular text styling here.

Reply:
Good catch! Fixed here bee9058d8061c4dabe2d96705c39b160749d9b61